project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
When I put jmx as a top-level dependency in tk-j9 and do `lein deps :tree` it shows up as `[org.clojure/java.jmx "0.2.0" :scope "test"]`

does this `:scope "test"` bit maybe have something to do with it?

If I removed jmx from the `dev` profile in tk-j9 (and kept it in as a normal dependency), then it didn't show up with this, and if I `lein install`ed it and used it in puppet-server then it worked.